<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <comment>
     Minimal Synthesizer for Qt applications
     Copyright (C) 2022, Pedro Lopez-Cabanillas &lt;plcl@users.sf.net&gt;

     This program is free software; you can redistribute it and/or modify
     it under the terms of the GNU General Public License as published by
     the Free Software Foundation; either version 3 of the License, or
     (at your option) any later version.

     This library is distributed in the hope that it will be useful,
     but WITHOUT ANY WARRANTY; without even the implied warranty of
     MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
     GNU General Public License for more details.

     You should have received a copy of the GNU General Public License
     along with this program. If not, see &lt;http://www.gnu.org/licenses/&gt;.
 </comment>
 <author>Pedro Lopez-Cabanillas</author>
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>490</width>
    <height>360</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>490</width>
    <height>360</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>490</width>
    <height>360</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Minimal Synthesizer</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>200</y>
      <width>50</width>
      <height>150</height>
     </rect>
    </property>
    <property name="text">
     <string>C</string>
    </property>
    <property name="autoExclusive">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_2">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>200</y>
      <width>50</width>
      <height>150</height>
     </rect>
    </property>
    <property name="text">
     <string>D</string>
    </property>
    <property name="autoExclusive">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_3">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>200</y>
      <width>50</width>
      <height>150</height>
     </rect>
    </property>
    <property name="text">
     <string>E</string>
    </property>
    <property name="autoExclusive">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_4">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>200</y>
      <width>50</width>
      <height>150</height>
     </rect>
    </property>
    <property name="text">
     <string>F</string>
    </property>
    <property name="autoExclusive">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_5">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>200</y>
      <width>50</width>
      <height>150</height>
     </rect>
    </property>
    <property name="text">
     <string>G</string>
    </property>
    <property name="autoExclusive">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_6">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>200</y>
      <width>50</width>
      <height>150</height>
     </rect>
    </property>
    <property name="text">
     <string>A</string>
    </property>
    <property name="autoExclusive">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_7">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>200</y>
      <width>50</width>
      <height>150</height>
     </rect>
    </property>
    <property name="text">
     <string>B</string>
    </property>
    <property name="autoExclusive">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_8">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>200</y>
      <width>50</width>
      <height>150</height>
     </rect>
    </property>
    <property name="text">
     <string>C'</string>
    </property>
    <property name="autoExclusive">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_9">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>40</y>
      <width>50</width>
      <height>150</height>
     </rect>
    </property>
    <property name="text">
     <string>C#</string>
    </property>
    <property name="autoExclusive">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_10">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>40</y>
      <width>50</width>
      <height>150</height>
     </rect>
    </property>
    <property name="text">
     <string>D#</string>
    </property>
    <property name="autoExclusive">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_11">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>40</y>
      <width>50</width>
      <height>150</height>
     </rect>
    </property>
    <property name="text">
     <string>F#</string>
    </property>
    <property name="autoExclusive">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_12">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>40</y>
      <width>50</width>
      <height>150</height>
     </rect>
    </property>
    <property name="text">
     <string>G#</string>
    </property>
    <property name="autoExclusive">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_13">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>40</y>
      <width>50</width>
      <height>150</height>
     </rect>
    </property>
    <property name="text">
     <string>A#</string>
    </property>
    <property name="autoExclusive">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QComboBox" name="deviceBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>301</width>
      <height>26</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>Audio Device</string>
    </property>
   </widget>
   <widget class="QSlider" name="volumeSlider">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>20</y>
      <width>161</width>
      <height>16</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>Volume</string>
    </property>
    <property name="maximum">
     <number>100</number>
    </property>
    <property name="singleStep">
     <number>10</number>
    </property>
    <property name="value">
     <number>90</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QSpinBox" name="bufferSpin">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>60</y>
      <width>81</width>
      <height>27</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>Audio Buffer Time</string>
    </property>
    <property name="suffix">
     <string> ms</string>
    </property>
    <property name="minimum">
     <number>10</number>
    </property>
    <property name="maximum">
     <number>200</number>
    </property>
    <property name="singleStep">
     <number>10</number>
    </property>
    <property name="value">
     <number>50</number>
    </property>
   </widget>
   <widget class="QSpinBox" name="octaveSpin">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>140</y>
      <width>81</width>
      <height>27</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>Octave Number</string>
    </property>
    <property name="prefix">
     <string>octave </string>
    </property>
    <property name="maximum">
     <number>9</number>
    </property>
    <property name="value">
     <number>3</number>
    </property>
   </widget>
  </widget>
 </widget>
 <tabstops>
  <tabstop>pushButton</tabstop>
  <tabstop>pushButton_9</tabstop>
  <tabstop>pushButton_2</tabstop>
  <tabstop>pushButton_10</tabstop>
  <tabstop>pushButton_3</tabstop>
  <tabstop>pushButton_4</tabstop>
  <tabstop>pushButton_11</tabstop>
  <tabstop>pushButton_5</tabstop>
  <tabstop>pushButton_12</tabstop>
  <tabstop>pushButton_6</tabstop>
  <tabstop>pushButton_13</tabstop>
  <tabstop>pushButton_7</tabstop>
  <tabstop>pushButton_8</tabstop>
  <tabstop>deviceBox</tabstop>
  <tabstop>volumeSlider</tabstop>
  <tabstop>bufferSpin</tabstop>
  <tabstop>octaveSpin</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
